test/reporter-html/reporter-html.js

```diff
@@ -44,6 +49,20 @@ QUnit.test( "running test name displayed", function( assert ) {
 	);
 } );
 
+QUnit.test( "previous test displays filtering run options", function( assert ) {
+	assert.expect( 2 );
+
+	var previous = this.getPreviousTest( assert ),
+		filteringLinks = previous.getElementsByTagName( "a" );
+
+	assert.ok( /^Rerun$/.test( filteringLinks[ 0 ].innerHTML ),
+		"'Rerun' link displayed"
+	);
+	assert.ok( /^Run module$/.test( filteringLinks[ 1 ].innerHTML ),
```

Comment:
Given that you're basically doing a full string compare, why not just use `assert.strictEqual( filteringLinks[ 1 ].innerHTML, "'Run module' link displayed" );`?